pennylane/devices/qubit/dq_interpreter.py

```diff
@@ -125,8 +125,7 @@ def interpret_measurement_eqn(self, eqn: "jax.core.JaxprEqn"):
 
     def interpret_measurement(self, measurement):
         # measurements can sometimes create intermediary mps, but those intermediaries will not work with capture enabled
-        disable()
-        try:
+        with stop_recording():
```

Comment:
Why is this even needed here? Even if intermediate ops/mps get created, during device execution, no one should be creating jaxpr. Is this for jitting purposes? So that we're not recording ops/meas while also recording arithmetic primitives associated with evolving/measuring a state vector?

Reply:
`process_samples` often creates intermediary measurements.

Reply:
Yes, but nothing would consume them since we're only executing what's in the original jaxpr

Reply:
Without this, we geT:
```
import pennylane as qml
import jax
qml.capture.enable()

@qml.qnode(qml.device('default.qubit', wires=1))
def circuit(x):
    qml.RX(x, 0)
    return qml.expval(qml.Z(0))

jax.jit(circuit)(0.5)
```
```
AttributeError: DynamicJaxprTracer has no attribute process_state
```

Reply:
😮 Thanks for checking!